docs/api/rundeck-api.md

```diff
@@ -22,7 +22,7 @@ Minimum
 Deprecation
 :   Future minimum version.
 
-For tips on getting started with the API, check out these [API Basics](/api/api_basics.md) .  For information on historical version changes please see [API Version History](/api/rundeck-api-versions.md).  Please note of any [incubating endpoints](/api/rundeck-api-versions.md#incubating_endpoints) that may be subject to change.
+For tips on getting started with the API, check out these [API Basics](/api/api_basics.md) .  For information on historical version changes please see [API Version History](/api/rundeck-api-versions.md). Check [swagger](/api/api-spec.md) Please note of any [incubating endpoints](/api/rundeck-api-versions.md#incubating_endpoints) that may be subject to change.
```

Comment:
should fix the link to openapi spec, rewrite as a sentence

Reply:
Fixed